resources/db-schema.cql

```diff
@@ -0,0 +1,144 @@
+CREATE TABLE forecast_names (
```

Comment:
ah, interesting, definitely. THis works for full breakdown and rebuild, but not for incremental migrations.